README.md

```diff
@@ -1 +1,19 @@
 # hw01
+## Sean Ennis Biography
+
+Hello Friends! My name is Sean Ennis and I am a third year **Public Policy Studies** major and am undecided if I want to pursue a **Joint BA** degree or a **Human Rights** minor _though I am running out of time to decide_.
+
+Some of my academic and non-academic interests are:
+
+* Academic
+  * Politics, I recommend [fivethirtyeight.com](https://fivethirtyeight.com)
```

Comment:
Great use of the link function, and it works perfectly! I also really liked the website, has a lot of good information on politics.